api/CookieChangeEvent.json

```diff
@@ -0,0 +1,197 @@
+{
+  "api": {
+    "CookieChangeEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CookieChangeEvent",
+        "support": {
+          "chrome": {
+            "version_added": "87"
```

Comment:
Confirmed by testing https://mdn-bcd-collector.appspot.com/tests/api/CookieChangeEvent on Chrome 86 and 87 (on Windows 10).